pkg/resmgr/cache/cache.go

```diff
@@ -367,6 +367,11 @@ type InsertContainerOption func(c *container)
 // Cache also saves its state upon changes to secondary storage and restores
 // itself upon startup.
 type Cache interface {
+	// EnableRDTControl is used to enable RDT class control.
+	EnableRDTControl(bool)
```

Comment:
super nit :roll_eyes: just caught my eye that it looks somewhat odd that Enable() func can be used to disable it. Not sure if `ConfigureRDTControl()`, `SetRDTControl()`, `RDTControlOnOff()` or smth would be any better though...

Reply:
Updated to use `ConfigureRDTControl()` (and `ConfigureBlockIOControl()`).